tests/jlm/tooling/Makefile.sub

```diff
@@ -0,0 +1,3 @@
+TESTS += \
+	jlm/tooling/TestJlcCommandGraphGenerator \
```

Comment:
Shouldn't these tests be JLC agnostic?
If so, then Jlc isn't suitable in the names.